src/sx-design/src/ProductCard.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import * as React from 'react';
+import sx from '@adeira/sx';
+
+import Heading from './Heading';
+
+export default function ProductCard(): React.Node {
+  // TODO: use Blurhash when loading the images
+
+  return (
+    <div className={styles('wrapper')}>
+      <Heading xstyle={styles.productTitle}>
+        <span>Product Name</span>
+      </Heading>
+    </div>
+  );
+}
+
+const styles = sx.create({
+  wrapper: {
+    height: 250,
+    width: 250,
+    display: 'flex',
+  },
+  productTitle: {
+    display: 'flex',
+    padding: 20,
+    color: 'var(--font-color-dark)',
+    transitionProperty: 'background-color',
+    transitionDuration: '500ms',
```

Comment:
This could potentially be a css variable too. 

Could be `--transition-duration-normal` , `--transition-duration-fast` etc.

Reply:
It's actually an error that `color` was using a CSS var (copy-paste mistake). SX has a different way how to achieve customizability which I'd say is the way to go for now: https://github.com/adeira/universe/tree/master/src/sx#composability-and-customizability